app/assets/stylesheets/registration.scss

```diff
@@ -127,22 +127,19 @@ font-weight: bold;
 
 }
 
-
-
 .logo {
   max-width: 100%;
   display: flex;
   flex-flow: row nowrap;
   align-items: center;
-
-
-
 }
 
 @media screen and (min-width: 480px) {
-
   form {
     max-width: 480px;
   }
+}
 
+.field_with_errors .error {
```

Comment:
`.field_with_errors` is the default class name added to an input field with an error by SimpleForm. This is configurable in the `simple_form.rb` initializer.